src/react-vrpc.js

```diff
@@ -1,8 +1,9 @@
-import React, { Component } from 'react'
+import React, { Component, useContext, useState, useEffect } from 'react'
```

Comment:
General comment on the new console.log statements added. I can see the warnings being useful to users but are all the normal logs needed? I can see it being frustrating to developers that are trying to debug their applications when they have so many logs that they cannot disable. Maybe we should make a debug argument that can enable these log messages but have it set to false by default.